### ISSUE TITLE
Containerd is not working behind a proxy

### ISSUE THREAD

**Issue by aojea** [kubernetes-sigs/kind#688]
**What happened**:

Containerd is not able to pull images if is working behind a proxy

**What you expected to happen**:

containerd should work behind a proxy

**How to reproduce it (as minimally and precisely as possible)**:

https://kubernetes.slack.com/archives/CEKK1KTN2/p1562335808184900

**Anything else we need to know?**:

The problem  is that we removed `/etc/systemd/system/containerd.service.d/http-proxy.conf`

when we merged this https://github.com/kubernetes-sigs/kind/pull/628 

**Environment:**

- kind version: (use `kind version`): > 0.4.0
- Kubernetes version: (use `kubectl version`):
- Docker version: (use `docker info`):
- OS (e.g. from `/etc/os-release`):

**Comment by aojea** [comment]
as a workaround 
```
[Service]    
Environment="HTTP_PROXY=${HTTP_PROXY:-}"    
Environment="HTTPS_PROXY=${HTTPS_PROXY:-}"    
Environment="NO_PROXY=${NO_PROXY:-localhost},${LOCAL_NETWORK}"    
EOF
```

@BenTheElder should we recover the `images/base/entrypoint` ?

**Comment by aojea** [comment]
Fixed by #694
